package.json

```diff
@@ -26,7 +26,7 @@
     "lint:js": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore \"**/*.{cjs,js,md,mjs}\"",
     "lint:prettier": "prettier --cache --cache-location .cache/prettier --cache-strategy content --check \"**/*.{cjs,js,json,md,mjs,scss,yaml,yml}\"",
     "lint:scss": "stylelint \"**/*.{md,scss}\"",
-    "check-links": "hyperlink --canonicalroot https://design-system.service.gov.uk/ --internal --recursive --source-maps deploy/public/index.html | tap-mocha-reporter min"
+    "check-links": "hyperlink --canonicalroot https://design-system.service.gov.uk/ --internal --recursive deploy/public/index.html | tap-mocha-reporter min"
```

Comment:
👍

Reply:
Odd that in the [spike to remove `--internal`](https://github.com/alphagov/govuk-design-system/pull/3174) there are still [source map errors](https://github.com/alphagov/govuk-design-system/actions/runs/6338651428/job/17216223025#step:6:3373) with this removed 🤷‍♂️